README.md

```diff
@@ -7,6 +7,7 @@
 * [Developer mode](#developer-mode)
 * [Enable trace support](#enable-trace-support)
 * [Enable debug console](#enable-debug-console)
+* [cpuset cgroup details](#cpuset-cgroup-details)
```

Comment:
nit, as you like nits....
`cpuset` is not capitalised, and the rest of the entries are - which may be as it is a 'proper command', in which case maybe make it a `code block style`.

Reply:
Yep - code block style looks best to me. Updated.